DSCResources/xUAC/xUAC.schema.psm1

```diff
@@ -20,7 +20,13 @@ Configuration xUac
             $ConsentPromptBehaviorAdmin = 2
             $EnableLua = 1
             $PromptOnSecureDesktop = 1
-        }    
+        }
+        "AlwaysNotifyAndAskForCredentials" 
+        {
```

Comment:
'AlwaysNotifyAndAskForCredentials' is missing from the validateset in the function.